openduck-py/openduck_py/routers/voice.py

```diff
@@ -486,5 +507,11 @@ async def connect_daily():
     await log_event(db, session_id, "ended_session")
 
 
+def run_connect_daily(room_url: str):
```

Comment:
since this runs in a new process, can you just do `asyncio.run(connect_daily(room=room_url))`? or does that not work